src/components/onboarding/OnBoardingModal/steps/Signer.tsx

```diff
@@ -0,0 +1,21 @@
+import { useSaveOnBoardingData } from 'src/rtk/features/onBoarding/onBoardingHooks'
+import OnBoardingContentContainer from '../OnBoardingContentContainer'
+import { OnBoardingContentProps } from '../types'
+
+export default function Signer(props: OnBoardingContentProps) {
+  const setProxySuccessMessage = useSaveOnBoardingData('signer')
```

Comment:
so the logic for all offchain signer, including the button, are all here only
and you can remove all excess props you added to the ContentContainer

Reply:
@teodorus-nathaniel Please check 77135d4